### DISCUSSION TITLE
Easy access to the list of software available in Neurodesk

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io#123] [Ideas]
In the end, the most important factor that will draw people to try Neurodesk is if their favourite tool is already included in the platform. Right now, it requires like 3 clicks to reach to this information, and it is not so evident where one should look for that.
I suggest to have a link to the current software list already in the main page (https://neurodesk.github.io/), in the top toolbar ("Software"), or in the neurocontainers page (https://neurodesk.github.io/docs/neurocontainers/).

Also -- the list itself is not nicely formatted (https://github.com/NeuroDesk/neurocommand/blob/main/cvmfs/log.txt). It's not a big deal, but I think that search engines do not catch it. Right now, when I search in Google "Neurodesk EEGLAB" for example, it bring me here: https://github.com/NeuroDesk/neurocommand/blob/main/neurodesk/apps.json
which is a bit (a lot) difficult to understand.

Lastly, if somebody arrives to the software list from a search engine, the list should have a title that indicates that it is CURRENTLY available in Neurodesk.

Let me know which of the above suggestions you like, and I can work on them.

**Comment by stebo85** [comment]
Including the list of software on the website would be great. It should be automatically generated based on https://github.com/NeuroDesk/neurocommand/blob/main/cvmfs/log.txt and nicely formatted. 

I think this should be possible in our current website similar to how we use the current version as a variable: https://github.com/NeuroDesk/neurodesk.github.io/blob/hugo-docsy/data/neurodesktop.toml

sudo docker run \
  --shm-size=1gb -it --privileged --name neurodesktop \
  -v ~/neurodesktop-storage:/neurodesktop-storage \
  -e HOST_UID="$(id -u)" -e HOST_GID="$(id -g)" \
  -p 8080:8080 -h neurodesktop-{{< params/neurodesktop/version >}} \
  vnmd/neurodesktop:{{< params/neurodesktop/version >}}


Looking forward to the contribution

cheers
Steffen

**Reply by aswinnarayanan** [reply]
@civier, sounds like an excellent idea.
Hugo docsy has some data displaying capabilities documented here: https://gohugo.io/templates/data-templates/
They may be useful for an implementation

**Comment by civier** [comment]
I like it that you like it.
I can definitely do it, but it will take some time (because I have a long
list of things I need to do beforehand, and I don't know the tools well).
Do you want to put it as a task in your Brainhack project? Or
alternatively, I can put it in our project (we are just finalising it now,
and will post it tomorrow the latest).

Oren

**Comment by stebo85** [comment]
sounds like a great hackathon project to me :)

**Comment by civier** [comment]
OK. So I count on you to add it to your hackathon, as you also know the
documentation tools better than myself.
If you find somebody that knows anything about search engines, even better
-- tell her to make sure the page is properly indexed by Google (maybe if
we include a link to the software website, this can help, but I truly don't
know how Google works these days).

I think you cannot modify your submission anymore, but I'll put the google
indexing requirement in our post, and if somebody presents herself, I'll
send her to you guys.

Cheers,
Oren
